src/trie.c

```diff
@@ -61,7 +59,6 @@ int add_node(char current, trie_t *t)
 
 }
 
-/* See trie.h */
 int insert_string(char *word, trie_t *t)
```

Comment:
See above comment about function naming and argument ordering.

Reply:
Will fix in separate issue